vms/vms/urls.py

```diff
@@ -5,6 +5,7 @@
 admin.autodiscover()
 
 urlpatterns = patterns('',
+    url(r'^$', include('home.urls', namespace='home')),
     url(r'^admin/', include(admin.site.urls)),
```

Comment:
> I will revert the settings.py change and include only this and create a
> different pull request.
> Is that Ok ?

Sure.

BTW you can avoid creating another PR by reverting and rebasing your changes. 

Why rebasing is encouraged? It helps new contributors to track down the commits easily. Contributor don't need to check unnecessary commits while tracking down a change. 

This will help you to learn some more git. Let me know if you face any issues while trying this.